roles/trivy/tasks/install-Debian-family.yml

```diff
@@ -1,45 +1,50 @@
 ---
-- name: Remove old architecture-dependent repository
-  become: true
-  ansible.builtin.apt_repository:
-    repo: "deb [ arch=amd64 ] https://aquasecurity.github.io/trivy-repo/deb {{ ansible_distribution_release }} main"
-    state: absent
-    filename: trivy
-  when: trivy_configure_repository|bool
+# Installation on Ubuntu 24.04 skipped, because the repository
+# 'https://aquasecurity.github.io/trivy-repo/deb noble Release'
+# does not have yet a release file.
+- name: Trivy installation
+  when: not ansible_distribution_version == '24.04'
+  block: # noqa
```

Comment:
Use `block:  # noqa osism-fqcn` here. Two spaces between `block:` and `# noqa osism-fqcn`.

Reply:
:+1: